### ISSUE TITLE
Project Structure Plan

### ISSUE THREAD

**Issue by OFRBG** [ethereum/web3-examples#3]
Instead of rushing to writing out examples, I'd like to ask @nivida if there is a plan/roadmap/napkin with notes to follow.

I can see that you want to have two main section: base packages like `truffle init` and usage examples. 

Let me know if you have a general idea, or if I should propose one, or start planning the sections. I see the repo is slightly lagging, so I'm willing to help maintain this. This is a topic of particular interest to me.

**Comment by nivida** [comment]
I think we should change the repostiory name to ``web3-documentation`` and we could create a folder structure like this: 
```
code-examples
  - eth
  - shh
 ....

documentation
  - eth
  - shh
  - net
  - bzz
  - utils
  - module-api

starter-projects
  - webpack
  - react
  - angular

```

We could change the repository name to ``web3-documentation``.

**Comment by kwikiel** [comment]
Yup, mixing every example into finished React app is unnecessary. 

What's important is showing how to use specific functions / libraries.

**Comment by nivida** [comment]
@kwikiel Yes, I think the examples should be as clean as possible without any JS framework. The starter projects should show a small example of a contract call or something else. Feel free to create a PR with the mentioned structure and don't hesitate to optimize it.
